### DISCUSSION TITLE
Updates on changing/securing Owner Address

### DISCUSSION THREAD

**Discussion by cwhiggins** [filecoin-project/lotus#5836] [Storage Provider] [answered]
When are we going to see some updates on being able to tie the owner address to a hardware wallet.  Right now Ledger wallet does not work.  Multisig is very complicated and not documented well, slack link below is for Multisig.

https://github.com/Zondax/ledger-filecoin/issues/65

https://filecoinproject.slack.com/archives/C01ERLPFDK2/p1610526148469900

**Comment by rjan90** [comment]
Agree! A more detailed guide on how to change the owner-address would be great, as well ledger-wallet support for that.

**Comment by LexLuthr** [comment, marked as answer]
Owner address can be updated.
https://lotu.sh/docs/storage-providers/addresses/#the-owner-address
